shared/profile/dumb.js

```diff
@@ -166,7 +166,7 @@ const propsBase: RenderProps = {
   onRevokeProof: proof => console.log('onRevokeProof', proof),
   parentProps: isMobile ? {} : {
     style: {
-      width: 640,
+      width: 720,
```

Comment:
This doesn't change what users will see, just what we see in this dev only preview